front/src/components/header/subcomponents/LoginModal.tsx

```diff
@@ -119,6 +122,58 @@ const LoginModal = ({ disclosureProps, setIsLoggedIn, unregisteredClearFollowedS
       });
   };
 
+  //  Handles OAuth login from Google
+  type Responses = GoogleLoginResponse | GoogleLoginResponseOffline;
+  const handleGoogleLogin = {
```

Comment:
Could we move this function and type into a new file? - something like `utils/google0Auth.ts`